lib/src/transformer/transformer.dart

```diff
@@ -68,8 +81,9 @@ class _TransformAssetWriter implements AssetWriter {
   _TransformAssetWriter(this.transform);
 
   @override
-  Future writeAsString(build.Asset asset, {Encoding encoding: UTF8}) async =>
-      transform.addOutput(_toBarbackAsset(asset));
+  Future writeAsString(build.Asset asset, {Encoding encoding: UTF8}) async {
```

Comment:
disclaimer: totally my opinion here. Since there is no inherently `async` operation here, I think it would be clearer to remove `async` and `return new Future.value(null);` here.

WDYT?